contracts/factory/src/contract.rs

```diff
@@ -69,26 +86,20 @@ impl FactoryTrait for Factory {
             &lp_init_info.stake_init_info,
         );
 
+        let lp_wasm_hash = get_config(&env).lp_wasm_hash;
+        let stake_wasm_hash = get_config(&env).stake_wasm_hash;
+        let token_wasm_hash = get_config(&env).token_wasm_hash;
```

Comment:
I know this is most likely optimized by the compiler, but nontheless - get config first and then use values from the config.

Reply:
I hope I have it now

Reply:
```Rust
        let config = get_config(&env);
        let lp_wasm_hash = config.lp_wasm_hash;
        let stake_wasm_hash = config.stake_wasm_hash;
        let token_wasm_hash = config.token_wasm_hash;
```